tuple/include/tuple_sketch.hpp

```diff
@@ -522,8 +558,7 @@ class compact_tuple_sketch: public tuple_sketch<Summary, Allocator> {
 
 };
 
-// builder
-
+// base builder
```

Comment:
intentional to have only `//` rather than `///`?

Reply:
I think we could expose this one as well